### PR TITLE
Update wikitemplate-IPFS.md

### PR DESCRIPTION
**Pull request by lidel**
This PR includes various tweaks / clarifications regarding IPFS test script

### CODE REVIEW

File: wikitemplate-IPFS.md [hunk 1]
```diff
@@ -167,30 +204,26 @@
 
 - [ ] Prerequisites: local node launched and local gateway configured. Go to `Settings -> IPFS`, there should be an available `Set up your IPNS keys` option, which opens `brave://settings/ipfs/keys`
 
-   ### `Sharing a local file using an IPFS key`
+   ### `Sharing a local file using an IPNS key`
 
-   - [ ] Verify you're able to share a local file via `IPFS > Share local file using IPFS`.
+   - [ ] Verify you're able to share a local file via `IPFS -> Share local file using IPNS -> self`.
```
Comment (lidel) [inline wikitemplate-IPFS.md:209]: This assumes UI change from https://github.com/brave/brave-browser/issues/16998 happens.

File: wikitemplate-IPFS.md [hunk 2]
```diff
@@ -87,68 +87,104 @@
...
-- [ ] Verify, on a new profile, you can load `https://en.wikipedia-on-ipfs.org`, switch `Method to resolve IPFS resources` to either `Gateway` or `Local node` in `brave://settings/ipfs`, and then see an `Open using IPFS` badge/icon in the URL bar.
-- [ ] Verify clicking on `Open using IPFS` on `https://blog.ipfs.io/24-uncensorable-wikipedia` loads `ipfs://bafybeiaieqdmhtnehaau7kqoj2lmdfqc7juk34cjyb7dxr35vahp22bquu/24-uncensorable-wikipedia/`.
+- [ ] Verify selecting `Use a public gateway` loads `https://cf-ipfs.com/ipns/en.wikipedia-on-ipfs.org/wiki/` which then redirect to unique origin at `https://en-wikipedia--on--ipfs-org.ipns.cf-ipfs.com/wiki/` 
+- [ ] Verify selecting `Use local node` loads `ipns://en.wikipedia-on-ipfs.org/wiki/`  
+
+### Public Path gateway
+
+- [ ] Verify, on a new profile, you can't change the IPFS public gateway address to `https://cloudflare-ipfs.com/` via `Settings -> IPFS -> IPFS public gateway address
+ -> Change`.
+- [ ] An error informing that only gateways with proper Origin isolation can be used in browser context is displayed.
```
Comment (lidel) [inline wikitemplate-IPFS.md:165]: This requires https://github.com/brave/brave-browser/issues/16999

File: wikitemplate-IPFS.md [hunk 3]
```diff
@@ -210,6 +243,8 @@
 - [ ] Verify the setting is enabled by default.
 - [ ] Verify disable/enable setting is retained between browser launch/restarts.
 - [ ] Verify setting state is retained during upgrade.
+- [ ] Verify you can't change the IPFS public gateway address to `https://cloudflare-ipfs.com/` (An error informing that only gateways with proper Origin isolation can be used in browser context is displayed)
+- [ ] Verify you you can change the IPFS public gateway address to `https://cf-ipfs.com/`  (provides origin isolation)
```
Comment (lidel) [inline wikitemplate-IPFS.md:247]: Requires https://github.com/brave/brave-browser/issues/16999

File: wikitemplate-IPFS.md [hunk 4]
```diff
@@ -87,68 +87,104 @@
...
+   ### `Sharing a local folder using IPFS (without IPNS keys)`
 
    - [ ] Go to `IPFS -> share Local Folder Using IPFS`, and select and import any local folder.
    - [ ] Verify the whole folder is downloaded and imported successfully.
    - [ ] Verify the import folder opens upon successful import.
    - [ ] Verify the shareable link is copied to the clipboard, and opens when pasting into a new tab.
 
+## "Open using IPFS" badge in address bar
+
+## BAdge shown only when IPFS support is enabled
```
Comment (srirambv) [inline wikitemplate-IPFS.md:100]: nit: `Badge` instead of `BAdge`

File: wikitemplate-IPFS.md (hunk 3 above)
Comment (srirambv) [reply wikitemplate-IPFS.md:247]: nit: repeating you

**Review by srirambv — CHANGES_REQUESTED** [review]

File: wikitemplate-IPFS.md (hunk 3 above)
Comment (stephendonner) [reply wikitemplate-IPFS.md:247]: Nit: I"m pedantic; can you include periods, just for consistency's sake, while you're in there?

Comment (stephendonner) [reply wikitemplate-IPFS.md:247]: you've got both `Origin isolation` and `origin isolation` here , just FYI

File: wikitemplate-IPFS.md (hunk 1 above)
Comment (stephendonner) [reply wikitemplate-IPFS.md:209]: Thanks; in that case, would it make sense to add a note indicating that, so we can track and remove later once landed?

File: wikitemplate-IPFS.md (hunk 2 above)
Comment (stephendonner) [reply wikitemplate-IPFS.md:165]: Same as above, re: would it make sense to add a note/reference to that issue, so we can keep track?

**Comment by stephendonner** [comment]
Thanks as always for the help and now this PR, @lidel 👍 keep 'em coming! 🙏

**Review by stephendonner — APPROVED** [review]
It. Looks.  Fantastic; thanks a mil!

🚢 ping it.

**Pull request merged**